### PR TITLE
Remove the end-of-lesson pilot flag

### PR DESCRIPTION
**Pull request by maureensturgeon**
Removes the `end-of-lesson-redirects` pilot flag to enable the end-of-lesson redirect experience. 

Screen Shot 2022-01-25 at 12 32 15 PM

There is also a small change to how the `completedLessonNumber` query param is handled, once the value is read the query param is removed so that it is not included in other links on the page.

## Links
PRs for the feature:
- https://github.com/code-dot-org/code-dot-org/pull/44394
- https://github.com/code-dot-org/code-dot-org/pull/44333

- jira ticket: [LP-2185](https://codedotorg.atlassian.net/browse/LP-2185)

## Testing story
Added a UI test and tested locally

## Follow-up work
Remove the end-of-lesson-redirects pilot from the pilots list

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jamescodeorg — APPROVED** [review]

File: apps/src/sites/studio/pages/scripts/show.js [hunk 1]
```diff
@@ -101,6 +102,9 @@ function initPage() {
   const mountPoint = document.createElement('div');
   $('.user-stats-block').prepend(mountPoint);
 
+  const completedLessonNumber = queryParams('completedLessonNumber');
+  updateQueryParam('completedLessonNumber', undefined);
```
Comment (maureensturgeon) [inline apps/src/sites/studio/pages/scripts/show.js:106]: We render links to levels on the unit overview page with the same query params as the page has, we don't want to maintain this query param after leaving this page, which is why I'm removing it here.

Comment (jamescodeorg) [reply apps/src/sites/studio/pages/scripts/show.js:106]: If we revisit this at some point, I think a comment in the code here would be helpful.  Also, is there a unit test to cover this behavior?  I can imagine accidentally breaking this.

Comment (maureensturgeon) [reply apps/src/sites/studio/pages/scripts/show.js:106]: No, we generally aren't unit testing our `show.js` files. But I do have an idea for a UI test that I could add, I'll do that today

Comment (jamescodeorg) [reply apps/src/sites/studio/pages/scripts/show.js:106]: Oh, darn it, I keep forgetting about that limitation.

**Pull request merged**